src/test/java/pages/BasePage.java

```diff
@@ -11,13 +11,14 @@
 
 public class BasePage {
 
-    WebDriver driver;
-    WebDriverWait wait;
+    public WebDriver driver;
+    public WebDriverWait wait;
     Actions actions;
```

Comment:
I recommend: make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

```
    protected Actions actions;
    protected WebDriverWait wait;
    protected WebDriver driver;
```

Reply:
I believe we haven't learn about protected access. So I don't know how to use it. Presentation slides for lesson 23 have the code for HW 22 and they use public instead of protected access. So I don't see how is it fair to take off points from my HW for that.